frontend/src/store/modules/v1/project.ts

```diff
@@ -153,10 +153,17 @@ export const useProjectV1Store = defineStore("project_v1", () => {
   };
 });
 
-export const useProjectV1List = (showDeleted: MaybeRef<boolean> = false) => {
+export const useProjectV1List = (
+  showDeleted: MaybeRef<boolean> = false,
+  forceUpdate = false
+) => {
   const store = useProjectV1Store();
   const ready = ref(false);
   watchEffect(() => {
+    if (!unref(forceUpdate)) {
```

Comment:
Should be `if(!unref(forceUpdate) && ready.value)` ?

Reply:
I'm afraid not. We make the initial call in `ProvideDashboardContext.vue`. So here it could be default lazy and nothing to do except `forceUpdate` is `true`